src/task1-3.js

```diff
@@ -0,0 +1,13 @@
+import csvtojson from 'csvtojson';
```

Comment:
I believe, we should not use TypeScript in first module, but anyway, script crashes with `SyntaxError: Cannot use import statement outside a module` exception

Reply:
This was meant to be the es6 modules syntax and not TypeScript, I'd love to use TypeScript in this, but I'm woefully unaware of it. I have the time to try, though, fwiw.